app/services/sorter.rb

```diff
@@ -0,0 +1,34 @@
+class Sorter
+  # NOTE: applications in a 'submitted' state can be assigned and therefore "In progresss", or unassigned.
+  # Unassigned applications require the visual label "Not assigned" for caseworker
+  STATUS_ORDER_CLAUSE = <<-SQL.squish.freeze
+  CASE WHEN state = 'submitted' THEN
+         CASE WHEN users.id IS NULL THEN  'not_assigned'
+         ELSE 'in_progress' END
+       WHEN state = 'further_info' THEN 'sent_back'
+       ELSE state END ?
+  SQL
+
+  ORDERS = {
+    'laa_reference' => "data->>'laa_reference' ?",
+    'firm_name' => "data->'firm_office'->>'name' ?",
+    'client_name' => "(data->'defendant'->>'first_name') || ' ' || (data->'defendant'->>'last_name') ?",
```

Comment:
so NSM records can have multiple defendants... :( so not 100% sure this works)

Reply:
Ah crap. I missed that. Thank you!

Reply:
Now fixed